src/docs/python-dependencies.mdx

```diff
@@ -13,10 +13,18 @@ So here are the rules we worked with so far:
 ## Rules on Dependencies
 
 1. Any new dependency needs to be thoroughly reviewed and approved
-2. Dependencies must be range pinned semver conforming in the requirements file of sentry
-3. Dependencies **must** be hard pinned in getsentry
+2. Dependencies must be range pinned semver conforming in the requirements file of **sentry**
+3. Dependencies **must** be hard pinned in **getsentry**
```

Comment:
Maybe make these links to repos or to requirements files themselves?

Reply:
ok, pushed to master 💪